lib/src/data/irma_repository.dart

```diff
@@ -116,7 +106,7 @@ class IrmaRepository {
     this.preferences,
     this.defaultKeyshareScheme,
   ) {
-    _inAppCredentialSubject.add(_InAppCredentialState());
+    _credentialStoreSubject.add(_CredentialStoreState());
```

Comment:
It would be a nice-to-have that credentials are deleted from this list when the expected credential is added, to make sure that this list does not increase size indefinitely. Most users won't kill their apps very often and then you might get weird behaviour if a credential that you obtained 3 days ago is still in this list.